feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SearchSessionsFragment.kt

```diff
@@ -82,6 +84,13 @@ class SearchSessionsFragment : DaggerFragment() {
         val groupAdapter = GroupAdapter<ViewHolder<*>>()
         binding.searchSessionRecycler.adapter = groupAdapter
 
+        binding.searchSessionRecycler.setOnApplyWindowInsetsListener { _, insets->
+            binding.searchSessionRecycler.updateLayoutParams<ConstraintLayout.LayoutParams> {
+                bottomMargin = insets.systemWindowInsetBottom
```

Comment:
Sorry for the issue.
Can you try padding and `android:clipToPadding="false"`?
Probably you can scroll to bottom. 👍 
https://github.com/DroidKaigi/conference-app-2020/blob/7e4a56cc15be5cca58c7f8c70d72ad810fecb344/feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml#L67